src/react/utils/open-in-browser.ts

```diff
@@ -0,0 +1,8 @@
+import electron from 'electron';
+
+function openInBrowser(link: string) {
+  electron.shell.openExternal(link);
```

Comment:
@Sansossio , we need to fix electron setup in order to expose Node's `fs` module, which is required to use `electron` API in the frontend